driver/netconf/capabilities.go

```diff
@@ -17,6 +17,14 @@ func (d *Driver) ServerHasCapability(s string) bool {
 	return false
 }
 
+// ServerCapabilities returns the list of capabilities the server
+// sent in the initial Hello message.
+func (d *Driver) ServerCapabilities() []string {
+	caps := make([]string, 0, len(d.serverCapabilities))
```

Comment:
Do we copy values and not returning the d.serverCapabilities because the original slice is mutated later on?